Sindri-Labs/sindri-resources
# Updating Halo2 sample circuit

This patch forces a downgrade of a rust package that does not like our current rust version.  Youll note that I added the `Cargo.lock` file to avoid this in the future.

## Testing Instructions:

Download either `mainnet_header.tar.gz` or `mainnet_header.zip` and upload to the frontend.  Ensure that compilation is successful. Alternatively, if you are logged into the CLI, you can pull this branch, switch directories to `circuit_database/halo2/axiom-v0.3.0/mainnet_header/` and call `sindri deploy`.  Compilation takes roughly 3-4 minutes.
